### ISSUE TITLE
remove sensitive data from the timeout log

### ISSUE THREAD

**Issue by haiyang1985** [apache/dubbo#5148]
- [x] I have searched the [issues](https://github.com/apache/dubbo/issues) of this repository and believe that this is not a duplicate.
- [x] I have checked the [FAQ](https://github.com/apache/dubbo/blob/master/FAQ.md) of this repository and believe that this is not a duplicate.

### Environment

* Dubbo version: 2.7.3
* Operating System version: macOs Sierra
* Java version: JDK 1.7

### Steps to reproduce this issue

1. sleep 1 second for the provider.
```
@Override
public String sayHello(String msg) {
  try {
    TimeUnit.SECONDS.sleep(1);
  } catch (Exception e) {
    e.printStackTrace();
  }
  return "welcome";
}
```
2. set 1 millisecond to trigger the timeout exception.
<dubbo:reference id="demoService" interface="xxx" timeout="1">
</dubbo:reference>

Pls. provide [GitHub address] to reproduce this issue.

### Expected Result
The log exception should be short and without extra sensitive data.

### Actual Result
The log exception contains the whole RpcInvocation, maybe caused sensitive data's leakage.

**Comment by htynkn** [comment]
Similar to #4151

**Comment by CrazyHZM** [comment]
Resolved in #5149
